web/themes/new_weather_theme/templates/block/block--weathergov-hourly-forecast.html.twig

```diff
@@ -0,0 +1,38 @@
+{%
+  set classes = [
+    'block',
+    'block-' ~ configuration.provider|clean_class,
+    'block-' ~ plugin_id|clean_class,
+  ]
+%}
+<div{{attributes.addClass(classes)}}>
+	{{ title_prefix }}
+	{% if label %}
+		<h2{{title_attributes}}>{{ label }}</h2>
+	{% endif %}
+	{{ title_suffix }}
+	{# Forecast:  hour, icon, condition desc (short), change of precip, temp degree, temp scale #}
+	<div class="grid-row flex-row minh-card">
+
+		{% set max_items = content["#data"]["max_items"] %}
+		{% for i in 1..max_items %}
```

Comment:
In the future, we can probably have the block code cap the number of results at the max, and then we can iterate over all of them here. Just a little less logic in the template. But we don't have results yet, so this is a good stopgap until then.